Refactor/project

Srinivas26k opened the pull request:
solved issue #42

Srinivas26k commented:
**Hi @supriya46788**,

I’ve updated the project structure and fixed file paths throughout the codebase.

Please review the changes and also check the **deployment path** accordingly, since the folder structure has changed.

Thanks!

supriya46788 commented:
please resolve the conlicts

supriya46788 commented:
did you run the repo in your local system? please do it if you haven't and index.html should be in the root folder... fix this 

<img width="1864" height="382" alt="image" src="https://github.com/user-attachments/assets/40685686-7350-4cff-92ae-01bfc41f2d98" />


<img width="1809" height="911" alt="image" src="https://github.com/user-attachments/assets/64bce752-d28c-4ea4-a7d0-dd751c866b1c" />

Srinivas26k commented:
**Update Summary**

* Moved `index.html` to the root directory.
* Resolved merge conflicts in `/html`, `/css`, and `/js`.
* Updated all file paths and imports across the project.

**Current Project Structure:**
```
├── .git/ 🚫 (auto-hidden)
├── assets/
│   ├── moon.png
│   └── sun.png
├── backend/
│   ├── server/
│   │   ├── models/
│   │   │   └── User.js
│   │   ├── routes/
│   │   │   └── auth.js
│   │   ├── bun.lock
│   │   ├── package-lock.json
│   │   ├── package.json
│   │   └── server.js
│   ├── .gitignore
│   └── README.md
├── css/
│   ├── about.css
│   ├── convert.css
│   ├── customize.css
│   ├── footer.css
│   ├── index.css
│   ├── login.css
│   ├── recipe.css
│   ├── scale.css
│   └── signup.css
├── favicon/
│   ├── android-chrome-192x192.png
│   ├── android-chrome-512x512.png
│   ├── apple-touch-icon.png
│   ├── favicon-16x16.png
│   ├── favicon-32x32.png
│   ├── favicon-48x48.png
│   ├── favicon.ico
│   └── site.webmanifest
├── html/
│   ├── about.html
│   ├── convert.html
│   ├── customize.html
│   ├── footer.html
│   ├── login.html
│   ├── recipe.html
│   ├── scale.html
│   └── signup.html
├── js/
│   ├── about.js
│   ├── convert.js
│   ├── customize.js
│   ├── index.js
│   ├── login.js
│   ├── recipe.js
│   ├── scale.js
│   └── signup.js
├── .env.example
├── .gitignore
├── CODE_OF_CONDUCT.md
├── CONTRIBUTING.md
├── License
├── README.md
├── firebase-config.js
└── index.html
```

**Next Steps:**
@supriya46788 I’d like to review the file structure and test it at the deployment level.
If you encounter any issues during testing, please let me know.

To streamline the process, we could set up **GitHub workflow automation** to verify code and preview deployments automatically. This will make it easier to track PRs and ensure everything works before merging.
